Remotion/Web/Development.WebTesting.ControlObjects/DropDownMenuControlObjectBase.cs

```diff
@@ -62,7 +62,7 @@ public bool IsOpen ()
     public void Close ()
     {
       if (IsOpen())
-        Context.Browser.Driver.ExecuteScript ("DropDownMenu_ClosePopUp()", Scope);
+        Context.Browser.Driver.ExecuteScript ("DropDownMenu_ClosePopUp(true)", Scope);
```

Comment:
isn't that a semantic change, passing true instead of false/undefined?

Reply:
And yes, it may be a valid bugfix, but this should then be a separate ticket.

Reply:
You're right, that wasn't the intention. There is no bug associated, I'll change this to `false` without further actions needed.

Reply:
@iam-mholle please create ticket (RMTEAM) for us to investigate if this actually a webtest bug. Thanks!